checker/src/main/java/org/checkerframework/checker/calledmethods/CalledMethodsTransfer.java

```diff
@@ -1,17 +1,41 @@
 package org.checkerframework.checker.calledmethods;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import javax.lang.model.element.AnnotationMirror;
+import javax.lang.model.type.TypeMirror;
+import org.checkerframework.checker.nullness.qual.Nullable;
 import org.checkerframework.common.accumulation.AccumulationTransfer;
+import org.checkerframework.dataflow.analysis.ConditionalTransferResult;
 import org.checkerframework.dataflow.analysis.TransferInput;
 import org.checkerframework.dataflow.analysis.TransferResult;
+import org.checkerframework.dataflow.cfg.block.ExceptionBlock;
 import org.checkerframework.dataflow.cfg.node.MethodInvocationNode;
 import org.checkerframework.dataflow.cfg.node.Node;
+import org.checkerframework.dataflow.expression.JavaExpression;
+import org.checkerframework.framework.flow.CFAbstractStore;
 import org.checkerframework.framework.flow.CFAnalysis;
 import org.checkerframework.framework.flow.CFStore;
 import org.checkerframework.framework.flow.CFValue;
+import org.checkerframework.javacutil.AnnotationUtils;
 
 /** A transfer function that accumulates the names of methods called. */
 public class CalledMethodsTransfer extends AccumulationTransfer {
 
+  /**
+   * {@link #makeExceptionalStores(MethodInvocationNode, TransferInput)} requires a TransferInput,
+   * but the actual exceptional stores need to be modified in {@link #accumulate(Node,
+   * TransferResult, String...)}, which only has access to a TransferResult. So this variable is set
+   * to non-null in {@link #visitMethodInvocation(MethodInvocationNode, TransferInput)} before the
```

Comment:
This is a helpful comment, thanks.  Could you further connect the dots by saying that a method will read the TransferInput from a CFStore?  (Or whatever will actually happen.)

Reply:
It's the other way around - the values in this map are `CFStore`s read from a `TransferInput`. I've expanded the comment to (hopefully) further clarify.